django/api/services/spreadsheet_uploader_prep.py

```diff
@@ -365,10 +363,15 @@ def validate_field_values(df, *columns, **kwargs):
             indices = []
             series = df[column]
             for index, value in series.items():
-                if str(value).upper() not in (item.upper() for item in allowed_values[column]) and value != '' and value is not None and not pd.isna(value):
-                    indices.append(index + kwargs.get("indices_offset", 0))
-                    if str(value) not in invalid_values:
-                        invalid_values.append(str(value))
+                items = [item.strip() for item in value.split(",")]
```

Comment:
Different datasets may have different delimiters (e.g. ; or |); I think we should pass a delimiter as part of kwargs. So perhaps something like: 
```
delimiter = kwargs.get("delimiter")
items = [value.strip()]
if delimiter is not None:
    items = [item.strip() for item in value.split(delimiter)]
```
And in `constants.py`, in the `CVP Data` dict of `DATASET_CONFIG`, you can add that as part of the `kwargs` field of `validation_functions`.